### PR TITLE
Adding reviewer details on reviewData object accessible in actions

### LINKED ISSUE #752: Include query from reviewers emails to facilitate sending correspondence on sendEmail action
2 days of of work considering cofig for: https://github.com/openmsupply/application-manager-server/issues/754

### PR DESCRIPTION
**Pull request by nmadruga**
fixes #752 

Needs testing!

### CODE REVIEW

File: src/components/graphQLConnect.ts [hunk 1]
```diff
@@ -49,6 +49,12 @@ class GraphQLdb {
           isLastLevel
           isLastStage
           status
+          reviewer {
+            username?: string
+            firstName?: string
+            lastName?: string
+            email?: string
+          }
```
Comment (CarlosNZ) [inline src/components/graphQLConnect.ts:57]:
```suggestion
          reviewer {
            id
            username
            firstName
            lastName
            email
          }
```
It's a query, not a type! 😆

File: src/components/graphQLConnect.ts [hunk 2]
```diff
@@ -75,6 +81,12 @@ class GraphQLdb {
             isLastLevel
             isLastStage
             status
+            reviewer {
+              username?: string
+              firstName?: string
+              lastName?: string
+              email?: string
+            }
```
Comment (CarlosNZ) [inline src/components/graphQLConnect.ts:89]:
```suggestion
            reviewer {
              id
              username
              firstName
              lastName
              email
            }
```

File: src/types.ts [hunk 3]
```diff
@@ -111,6 +111,12 @@ export interface ActionApplicationData {
     levelNumber?: number
     isLastLevel?: boolean
     status?: string
+    reviewer: {
+      username?: string
+      firstName?: string
+      lastName?: string
+      email?: string
+    }
```
Comment (CarlosNZ) [inline src/types.ts:119]:
```
    reviewer?: {
      id: number
      username: string
      firstName: string
      lastName: string
      email: string
    }
```
Wouldn't it be?

**Review by CarlosNZ — CHANGES_REQUESTED** [review]
See comments.

Also -- I'd put the `reviewerId` in there too :)

File: src/components/graphQLConnect.ts (hunk 1 above)
Comment (nmadruga) [reply src/components/graphQLConnect.ts:57]: Hhahaha bad copy! :P

File: src/types.ts (hunk 3 above)
Comment (nmadruga) [reply src/types.ts:119]: Yea, makes sense!

**Review by CarlosNZ — APPROVED** [review]
Great, it returns what it's supposed to when testing. 👍

Screen Shot 2022-09-13 at 4 07 03 PM

**Pull request merged**